### PR TITLE
Feature - aircraft  true course 

### PR DESCRIPTION
**Pull request by mclarke2**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: regression/scripts/VTOL/test_Tiltwing.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def main():
     plot_mission(results)   
 
     # save, load and plot old results 
-    #save_tiltwing_results(results)
+    save_tiltwing_results(results)
```
Comment (rachealerhard) [inline regression/scripts/VTOL/test_Tiltwing.py:53]: comment out

File: trunk/SUAVE/Methods/Power/Battery/Sizing/initialize_from_circuit_configuration.py [hunk 2]
```diff
@@ -50,8 +50,8 @@ def initialize_from_circuit_configuration(battery,module_weight_factor = 1.42):
     nominal_voltage              = battery.cell.nominal_voltage       
     total_battery_assemply_mass  = battery.cell.mass * battery.pack_config.series * battery.pack_config.parallel   
     battery.mass_properties.mass = total_battery_assemply_mass*module_weight_factor  
-    battery.specific_energy      = (amp_hour_rating*nominal_voltage)/battery.cell.mass  * Units.Wh/Units.kg   
-    battery.max_energy           = total_battery_assemply_mass*battery.specific_energy    
+    battery.specific_energy      = (amp_hour_rating*nominal_voltage)/battery.cell.mass   
+    battery.max_energy           = total_battery_assemply_mass*battery.specific_energy*(Units.Wh/Units.kg)
```
Comment (rachealerhard) [inline trunk/SUAVE/Methods/Power/Battery/Sizing/initialize_from_circuit_configuration.py:54]: I think this should be returned to the way it was; battery specific energy is typically J/kg, so I think the change you made here mixes up the units

File: trunk/SUAVE/Plots/Performance/Mission_Plots.py [hunk 3]
```diff
@@ -843,6 +841,77 @@ def plot_flight_conditions(results, line_color = 'bo-', save_figure = False, sav
     return
 
 # ------------------------------------------------------------------
+#  Aircraft Trajectory
+# ------------------------------------------------------------------
+## @ingroup Plots
+def plot_flight_trajectory(results, line_color = 'bo-', line_color2 = 'rs--', save_figure = False, save_filename = "Flight_Trajectory", file_type = ".png"):
+    """This plots the flights the conditions
```
Comment (rachealerhard) [inline trunk/SUAVE/Plots/Performance/Mission_Plots.py:848]: reword

**Review by rachealerhard — APPROVED** [review]
see minor comments

File: regression/scripts/VTOL/test_Tiltwing.py (hunk 1 above)
Comment (mclarke2) [reply regression/scripts/VTOL/test_Tiltwing.py]: Commented out

File: trunk/SUAVE/Methods/Power/Battery/Sizing/initialize_from_circuit_configuration.py (hunk 2 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Power/Battery/Sizing/initialize_from_circuit_configuration.py:54]: Changed back

File: trunk/SUAVE/Plots/Performance/Mission_Plots.py (hunk 3 above)
Comment (mclarke2) [reply trunk/SUAVE/Plots/Performance/Mission_Plots.py:848]: Fixed

**Pull request merged**